tools/torchfix/torchfix/visitors/vision/__init__.py

```diff
@@ -150,6 +150,27 @@ class TorchVisionDeprecatedPretrainedVisitor(TorchVisitor):
     }
     # fmt: on
 
+    # The same model can be imported from torchvision.models directly,
+    # or from a submodule like torchvision.models.resnet.
+    MODEL_SUBMODULES = (
```

Comment:
Isn't it a bit problematic, as it makes assumption one is using latest torchvision?

Reply:
Everything in TorchFix is about latest currently.
In future I want to add a configuration to specify target versions.
But that particular API change was introduced in TorchVision 0.15, one release before the current.